tempesta_fw/http.h

```diff
@@ -412,5 +415,10 @@ void tfw_msg_add_data(void *handle, TfwMsg *msg, char *data, size_t len);
 int tfw_http_sess_obtain(TfwHttpReq *req);
 int tfw_http_sess_resp_process(TfwHttpResp *resp, TfwHttpReq *req);
 void tfw_http_sess_put(TfwHttpSess *sess);
+extern bool tfw_cfg_sticky_sessions;
+extern bool tfw_cfg_sticky_sessions_failover;
+TfwConnection *tfw_http_sess_get_conn(TfwHttpReq *req, TfwSrvGroup *sg);
+int tfw_http_sess_save_conn(TfwHttpReq *req, TfwSrvGroup *sg,
+			    TfwConnection *conn);
```

Comment:
As for me, it is "just good enough". Moving all the session related types and functions lead to circular dependencies: `TfwHttpReq` require `TfwHttpSess`, but all the session related functions require `TfwHttpReq`. Both approaches using forward declarations or moving only part of session related staff to separate header don't look good for me. How do you think?